### PR TITLE
Seafile: version bump to 7.0.4

### PR DESCRIPTION
**Pull request by evdenis**
Version bump for seafile && seafile-client.

### CODE REVIEW

File: net-misc/seafile/seafile-7.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```
Comment (juippis) [inline net-misc/seafile/seafile-7.0.4.ebuild]: All good, but we are currently looking into dropping python2.7 support and I see upstream for this package has enabled python3 support. Could you look into updating to that?
https://github.com/haiwen/seafile/commit/a9252aca234bd44bd4ae579e0d045d150c6751a6

Comment (evdenis) [reply net-misc/seafile/seafile-7.0.4.ebuild]: Done.

File: net-libs/libsearpc/libsearpc-3.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}
+	>=dev-libs/glib-2.26.0
+	>=dev-libs/jansson-2.2.1
+	>=sys-devel/automake-1.16.1-r2:1.16"
```
Comment (juippis) [inline net-libs/libsearpc/libsearpc-3.2.0.ebuild]: There's a better way to do this using autotools.eclass, I'll fix it in a separate commit. You should become a maintainer for this package though ;)

Comment (evdenis) [reply net-libs/libsearpc/libsearpc-3.2.0.ebuild]: Well, you can add me with your fix or I will do it in the next release.

Comment (evdenis) [reply net-libs/libsearpc/libsearpc-3.2.0.ebuild]: I took the information from this page https://wiki.gentoo.org/wiki/Project:Python/Byte_compiling Probably you could update it with proper autotools.eclass fix. Thanks!

Comment (juippis) [reply net-libs/libsearpc/libsearpc-3.2.0.ebuild]: > Well, you can add me with your fix or I will do it in the next release.

Please do it next time :)

**Pull request closed without merging**